app/main.ipynb

```diff
@@ -126,19 +158,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[1, 2, 4, 5]]\n",
+    "result = np.sum(selected_elements)\n",
+    "\n",
+    "print(\"selected_elements:\", selected_elements)\n",
+    "print(\"selected_elements_total:\", result)"
```

Comment:
The sum of the selected elements is expected to be 27, but the current implementation results in 26.3. Please ensure the correct indices are used to achieve the desired sum.